crates/re_types_builder/src/codegen/docs/mod.rs

```diff
@@ -173,6 +172,16 @@ fn object_page(reporter: &Reporter, object: &Object, object_map: &ObjectMap) ->
             object.kind.plural_snake_case(),
             object.name
         );
+        putln!(
+            page,
+            // `_1` is doxygen's replacement for ':'
+            // https://github.com/doxygen/doxygen/blob/Release_1_9_8/src/util.cpp#L3532
+            " * 🌊 [C++ API docs for `{}`](https://ref.rerun.io/docs/cpp/stable/structrerun_1_1{}_1_1{}.html{})",
```

Comment:
Let's sort the languages lexicographically instead of… whatever order this is :)

Reply:
The order is also weird here:

<img width="192" alt="image" src="https://github.com/rerun-io/rerun/assets/1148717/83a83bc0-aab0-4549-93ed-3cccdd923984">

Reply:
Python, C++, Rust is the order we use pretty much everywhere

Reply:
Far from everywhere:

<img width="386" alt="image" src="https://github.com/rerun-io/rerun/assets/1148717/f249b6d3-9b5a-4aa4-a626-30823de93beb">

<img width="293" alt="Screenshot 2023-11-27 at 12 27 41" src="https://github.com/rerun-io/rerun/assets/1148717/ce2b2d4f-f3bb-4276-9ffe-9acd3c835dfa">

<img width="279" alt="image" src="https://github.com/rerun-io/rerun/assets/1148717/fed24449-9c6c-4d52-8240-96e8daa912e4">

Reply:
As the number of supported languages grows, anything but lexicographical order will just be weird